File: src/Containers/Answer.java
package Containers;

public class Answer extends Post{

    private int questionId;

    //posttype = 2
    public Answer(int id, String dateCreated, int score, String title, String body, int ownerId, int questionId) {
        super(id, dateCreated, score, title, body, ownerId);
        this.questionId = questionId;
    }

    public int getQuestionId() {
        return questionId;
    }

    @Override
    public String toString() {
        String superString =  super.toString();
        return String.format("%s , questionId : %d}",
                superString.substring(0, superString.length() - 2), questionId);
    }
}

File: common/models/answer.py
from common.models.post import Post

class Answer(Post):
    def __init__(self, post_id, site_post_id, dateCreated, score, title, body, ownerId, questionId):
        super().__init__(post_id, site_post_id, dateCreated, score, title, body, ownerId)
        self.questionId = questionId

    def __str__(self):
        return f"{str(super())[:-2]}, questionId: {self.questionId}"
